src/DnsRootStack.ts

```diff
@@ -155,6 +155,14 @@ export class DnsRootStack extends cdk.Stack {
       domainName: '4bd2d704cc5f72f39cf605cf24500bde.13f6be6ddcec328123708fd74b620d25.comodoca.com.',
     });
 
+    /**
+     * DocPoc Sectigo
+     */
+    new Route53.CnameRecord(this, 'docpoc-validation-record-accp', { // DocPoc accp - esb (docpoc-accp.csp-nijmegen.nl)
+      zone: this.cspNijmegenZone,
+      recordName: '_C09AF98DC2FE54695443BF987E37C329',
+      domainName: '98DCDAD749B9B0C55544167D8CE5202C.2F575C50C0E49874844BC0A5A49CAD06.sectigo.com',
```

Comment:
Niet helemaal zeker wat aws doet, maar de . Op het einde is mogelijk noodzakelijk.

Reply:
Punt toegevoegd. Ook in andere PR voor main